### PR TITLE
IAM Client Library for Python

### PR DESCRIPTION
**Pull request by Landrito**

### CODE REVIEW

File: generated/python/gapic-google-cloud-iam-v1/README.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
+GAPIC library for the Google Admin API
```
Comment (bjwatson) [inline generated/python/gapic-google-cloud-iam-v1/README.rst]: Hmm... this should be hand-edited to say "Google IAM Admin API". We have a [Github issue](https://github.com/googleapis/toolkit/issues/270) to improve where we source this title (e.g. from https://github.com/googleapis/googleapis/pull/151/files#diff-b8750c321eaa4d100c91edbf078d08a3R7).

Comment (Landrito) [inline generated/python/gapic-google-cloud-iam-v1/README.rst]: Ah I see. Fixed for this case.

File: generated/python/gapic-google-cloud-iam-v1/docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,318 @@
+# -*- coding: utf-8 -*-
+#
+# gax-google-iam-admin-v1 documentation build configuration file
```
Comment (Landrito) [inline generated/python/gapic-google-cloud-iam-v1/docs/conf.py]: Should be fixed with the re-gen

**Comment by bjwatson** [comment]
LGTM

**Comment by bjwatson** [comment]
@Landrito Please note that I generated a `0.10.0` version of the gRPC IAM Admin dependency in https://github.com/googleapis/api-client-staging/pull/24.

**Pull request merged**